pkg/action/scan.go

```diff
@@ -147,12 +155,35 @@ func errIfHitOrMiss(frs map[string]*bincapz.FileReport, kind string, scanPath st
 	return nil
 }
 
+// structs and types for sorting reports alphabetically by path
+// kv stores the keys and values of a map.
+type kv struct {
+	key   string
+	value *bincapz.FileReport
+}
+
+// reports is a slice of paths and their respective file reports.
+type reports []kv
```

Comment:
I don't think this type makes the code any clearer, can it be removed? `sort.Slice()` doesn't require special types for sorting.

If we need it for readability, it could definitely use a clearer name, but in my experience, type aliases for slices hurt readability.

Reply:
Removed in [`c03d232` (#405)](https://github.com/chainguard-dev/bincapz/pull/405/commits/c03d232eece70d75d3ed81466e19bfbdb8af0423).